src/cdk/schematics/migration.json

```diff
@@ -26,6 +26,12 @@
       "description": "Updates the Angular CDK to v10",
       "factory": "./ng-update/index#updateToV10"
     },
+    "migration-v11": {
```

Comment:
Note that I'm not totally sure whether this will prevent people from running it, I'm hoping that's what the `private: true` below will do. If that's not the case, I'm fine with leaving out the `migration.json` files for later since we usually get blocked on the TS changes.

Reply:
What is the motivation for adding `private` at all here? Wouldn't it be okay to just have this guarded by the version. Since we are not at v11, the migration will not run at all.

Reply:
I'm concerned we'd rather forget removing `private` in v11 then. Should be fine just having it guarded by the version.

Reply:
I'm not totally familiar with where this shows up. My concern was that it would be printed out somewhere as a list of available migrations, but if you think that it won't, I'll get rid of it.

Reply:
I see. Yeah, I have not seen the migration list being printed out anywhere. I think we should be good just removing this.